src/Types/Json.php

```diff
@@ -2,52 +2,17 @@
 
 namespace LaravelDoctrine\ORM\Types;
 
-use Doctrine\DBAL\Platforms\AbstractPlatform;
-use Doctrine\DBAL\Types\JsonArrayType;
+use Doctrine\DBAL\Types\JsonType;
 
 /**
- * Custom Doctrine data type for JSON.
- * Doctrine has a json_array type but, as its name suggests, it was designed with
- * only arrays in mind. This extending type fixes a bug with the json_array type
- * wherein a null value in database gets converted to an empty array.
- * IMPORTANT NOTE: you must register custom types with Doctrine:
- *      \Doctrine\DBAL\Types\Type::addType('json', '\Path\To\Custom\Type\Json');
- * @link http://www.doctrine-project.org/jira/browse/DBAL-446
- * @link https://github.com/doctrine/dbal/pull/655
+ * @deprecated Extend JsonType directly
+ *
+ * This type was added when Doctrine only had `JsonArrayType`.
+ * `JsonArrayType` have a flaw with empty/null values.
```

Comment:
I think this should be 'had' instead of 'have'.
```suggestion
 * `JsonArrayType` had a flaw with empty/null values.
```

Reply:
My english is not good, so you might be right, but JsonArrayType (`json_array`) does still have this issue _(though its deprecated)_.

Reply:
I have changed `have` to `had` now.